libsolidity/ast/Types.cpp

```diff
@@ -2292,9 +2287,9 @@ string FunctionType::toString(bool _short) const
 	for (auto it = m_parameterTypes.begin(); it != m_parameterTypes.end(); ++it)
 		name += (*it)->toString(_short) + (it + 1 == m_parameterTypes.end() ? "" : ",");
 	name += ")";
-	if (m_isConstant)
+	if (isConstant())
 		name += " constant";
```

Comment:
What is this `toString` function used for? If we change the parser to support `view` next to `constant`, but only print `view`, would that had any adverse effect? I think it is only used in error reporting.